packages/common/src/index.ts

```diff
@@ -141,13 +141,25 @@ export interface CommonOpts extends BaseOpts {
    * const common = new Common({ chain: 'myCustomChain1', customChains: [ myCustomChain1 ]})
    * ```
    *
-   * Pattern 2 (with genesis state, see {@link GenesisState} for format):
+   * Pattern 2 (with genesis state see {@link GenesisState} for format). Note that in {@link AccountState} there are two
+   * accepted types. This allows to easily insert accounts in the genesis state
```

Comment:
I think instead of this additional note here, you can remove and add some more context below with a `Pattern 3 (with complex genesis state, containing contract accounts and storage):` for the next example below

```suggestion
   * Pattern 2 (with genesis state see {@link GenesisState} for format):
```

Reply:
Just updated it! Please let me know if you have any other feedback and I can implement it - Once everything is cool you can resolve this @ryanio :-D